test/kitchen/test/integration/common/rspec/spec_helper.rb

```diff
@@ -197,7 +197,15 @@ def is_port_bound(port)
 
   it 'is properly signed' do
     if os == :windows
-      msi_path = 'C:\\Users\\azure\\AppData\\Local\\Temp\\kitchen\\cache\\ddagent-cli.msi'
+      msi_path_base = 'C:\\Users\\'
+      msi_path_end = '\\AppData\\Local\\Temp\\kitchen\\cache\\ddagent-cli.msi'
+      msi_path_azure = msi_path_base + 'azure' + msi_path_end
+      msi_path_datadog = msi_path_base + 'datadog' + msi_path_end
+      if File.exist?(msi_path_azure)
+        msi_path = msi_path_azure
+      else
+        msi_path = msi_path_datadog
+      end
```

Comment:
what's this block of changes for? (a comment to explain why we do this would be perfect)

Reply:
Sure! Basically I changed the username when I made a better password, the one I had before was azure, and I wanna make this more fault tolerant by also allowing the default username to be used without it breaking. The error message is really horrible and hard to parse, so I wanna avoid anyone having to parse it.